lib/events/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "events",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "jest",
+    "build": "tsc",
+    "lint": "tslint --project tsconfig.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/libero/reviewer"
+  },
+  "author": "Nathan Lisgo",
+  "license": "ISC",
+  "dependencies": {
+    "@libero/event-bus": "file:../event-bus"
```

Comment:
The `file:../event-bus` reference means that the path is relative to this `package.json` file. We should perhaps use this convention more widely?